VisualC-WinRT/SDL-UWP.vcxproj

```diff
@@ -280,6 +280,7 @@
       <PrecompiledHeaderOutputFile Condition="'$(Configuration)|$(Platform)'=='Release|x64'">$(IntDir)$(TargetName)_cpp.pch</PrecompiledHeaderOutputFile>
     </ClCompile>
     <ClCompile Include="..\src\cpuinfo\SDL_cpuinfo.c" />
+    <ClCompile Include="..\src\dialog\windows\SDL_dialog.c">

```

Comment:
This is probably wrong (sorry), and we need to use the dummy implementation for UWP (WINDOWS_STORE in cmake)